nettacker/config.py

```diff
@@ -145,6 +146,11 @@ class DefaultSettings(ConfigBase):
     usernames_list = None
     verbose_event = False
     verbose_mode = False
+    scan_compare_id = None
+    compare_report_path_filename = "/results_{date_time}_{random_chars}.json".format(
```

Comment:
@Captain-T2004  shouldn't this be {results_dir} instead of "/results"...?

Reply:
@securestep9, actually I added the results directory path as a PathConfig object variable. And also here only the name of the file is being generated instead of the complete path. Inside my create_compare_report method in graph.py, I have included the basepath from an PathConfig object and then created a complete path from user input + basepath in PathConfig. This is why I removed {results_dir} from here. The error I was facing was that each time a comparison is ran through API the default file name that was generated here was being rewritten as a new filename was not generated each time. In order to generate a new file name each time I have added some helper functions in /utils/common.py and along with the base filepath from PathConfig I can create a complete path inside my create_compare_report method. If you think this is incorrect do let me know I will reverse it.

Reply:
@Captain-T2004 Please use  {results_dir}  - this is the only folder which shold be for Nettacker results. The problem with file name conflict / files being overwritten can be solved by adding strings to the file name e.g. 'report_compare_{scan_id}' as scan_is is already random it actually makes sense to use scan_id as part of the results filename to identify the results belonging to a specific scan